tests/bcl-test/BCLTests/common-CorlibTests.ignore

```diff
@@ -0,0 +1,345 @@
+# crashes:
```

Comment:
Anything that crashes should also have a corresponding bug in the mono repo.

Reply:
Issue added: https://github.com/mono/mono/issues/12170